stdlib/encodings/charmap.pyi

```diff
@@ -0,0 +1,29 @@
+import codecs
+from _codecs import _CharMap
+from _typeshed import ReadableBuffer
+
+class Codec(codecs.Codec):
+    encode = codecs.charmap_encode  # type: ignore[assignment]  # pyright: ignore[reportAssignmentType]
+    decode = codecs.charmap_decode  # type: ignore[assignment]  # pyright: ignore[reportAssignmentType]
+
+class IncrementalEncoder(codecs.IncrementalEncoder):
+    mapping: _CharMap
```

Comment:
This can also be None I believe. Same for the other mapping attributes in this file.